platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -182,7 +183,7 @@ void doImport() {
         } else if (version.equals(migratable.getVersion())) {
           migratable.doImport(this);
         } else {
-          migratable.doIncompatibleImport(this, version);
+          migratable.doVersionUpgradeImport(this, migrationProps, version);
```

Comment:
❗️ `version` is already part of the context passed to the migratable. We should simply add it to the `ImportMigrationContext` interface as opposed to pass it through here.